frame/staking/reward-curve/src/lib.rs

```diff
@@ -205,14 +205,14 @@ impl Parse for INposInput {
 		})?;
 		<syn::Token![,]>::parse(&args_input)?;
 		let ideal_stake = parse_field::<keyword::ideal_stake>(&args_input, Bounds {
-			min: 0_100_000,
+			min: 100_000,
```

Comment:
considering all those unit are expressed in per_million the old syntax was more clear to me

Reply:
undone those changes.